compiler/aarch64/codegen/OMRTreeEvaluator.hpp

```diff
@@ -49,6 +49,14 @@ namespace TR { class CodeGenerator; }
  */
 TR::Register *genericReturnEvaluator(TR::Node *node, TR::RealRegister::RegNum rnum, TR_RegisterKinds rk, TR_ReturnInfo i, TR::CodeGenerator *cg);
 
+/**
+ * @brief Helper function for Null/Resolve CHK Evaluators
+ * @param[in] node : node
+ * @param[in] needsResolve : if node needs resoloution
+ * @param[in] cg : CodeGenerator
+ */
+TR::Register *evaluateNULLCHKWithPossibleResolve(TR::Node *node, bool needsResolve, TR::CodeGenerator *cg);
```

Comment:
I don't think you need to declare this function in OMRTreeEvaluator.hpp.
Are you planning to call `evaluateNULLCHKWithPossibleResolve()` from files other than ControlFlowEvaluator.cpp?